pulse_actions/publisher.py

```diff
@@ -22,8 +24,7 @@ class MessageHandler:
 
     def __init__(self):
         """Create Publisher."""
-        user  = os.environ.get('PULSE_USER')
-        password = os.environ.get('PULSE_PW')
+        user, password = get_user_and_password()
```

Comment:
publisher.py is called from the handlers modules that are called from worker.py, so with the current structure it is impossible to get to this point without the appropriate env variables set up. But since we now have get_user_and_password it is cleaner to use it here.